languages/ruby/lib/oso/polar/ffi/query.rb

```diff
@@ -74,6 +75,15 @@ def process_messages
             message.process
           end
         end
+
+        # @return [String]
```

Comment:
This doesn't look like it returns a string -- did you mean to return `res.to_s`?